src/main/java/net/fabricmc/installer/client/ClientHandler.java

```diff
@@ -52,13 +57,29 @@ public void install() {
 				if (createProfile.isSelected()) {
 					ProfileInstaller.setupProfile(mcPath, profileName, gameVersion);
 				}
+				SwingUtilities.invokeLater(() -> showInstalledMessage(loaderVersion, gameVersion));
 			} catch (Exception e) {
 				error(e);
 			}
 			buttonInstall.setEnabled(true);
 		}).start();
 	}
 
+	private void showInstalledMessage(String loaderVersion, String gameVersion) {
+		JEditorPane pane = new JEditorPane("text/html", new MessageFormat(Utils.BUNDLE.getString("prompt.install.successful")).format(new Object[]{loaderVersion, gameVersion, Reference.fabricApiUrl}));
+		pane.setEditable(false);
+		pane.addHyperlinkListener(e -> {
+			if (e.getEventType() == HyperlinkEvent.EventType.ACTIVATED) {
+				try {
+					Desktop.getDesktop().browse(e.getURL().toURI());
```

Comment:
pretty sure this will crash on MacOS

Reply:
This needs testing, not sure why it would though.

Reply:
I didn't crash on macOS, but the font and background color doesn't match the rest of the box:

![macOS Fabric API install popup](https://user-images.githubusercontent.com/24855774/82169022-2dd60480-988e-11ea-99f5-156f2622bfd4.png)

Reply:
Oh yeah I have to set the font, give me a second.

Reply:
The formatting issues have been fixed in 9359e301a7e53ead97055cf02ed5f5545e141dad, so now macOS support is complete:

![macOS Fabric API install popup with correct formatting](https://user-images.githubusercontent.com/24855774/82172795-3a138f00-9899-11ea-9289-5a33ec7748e9.png)